app/views/unattended/provisioning_templates/PXEGrub/jumpstart_default_pxegrub.erb

```diff
@@ -4,7 +4,10 @@ name: Jumpstart default PXEGrub
 model: ProvisioningTemplate
 oses:
 - Solaris
-%>
+description: |
+  The template to render PXEGrub bootloader configuration for Solaris.
+  The output is deployed on the host's subnet TFTP proxy.
+-%>
```

Comment:
Maybe adding something like this to all PXEGrub templates:

This template is for legacy Grub 1.x, you are probably looking for Grub2.